packages/frontend/src/components/Dialog/styles.module.scss

```
@@ -4,6 +4,10 @@ $paddingVertical: 20px;
 
 .dialog {
   padding: 0;
+  @media (width < 500px){
+    width: 100vw !important;
+    min-width: 100vw !important;
```

```suggestion
    max-width: 100vw !important;
    min-width: 0 !important;
```

This makes more sense semantically, and also doesn't make the dialogs that are narrower than 500px initially stretch unnecessarily.